packages/ui/Media/Media.js

```diff
@@ -312,7 +282,7 @@ class Media extends React.Component {
 				{...this.handledMediaEvents}
 				ref={this.mediaRef}
 			>
-				{source}
+				{children && children.type === 'source' ? children : source}
```

Comment:
I think we need to revisit this logic. I'd suggest favoring `source` over `children` since it is more explicit in its purpose. Also, I'd suggest dropping the `type` check since it's possible that authors might want to abstract source selection behind a custom component (e.g. passing a generic path which the component resolves to the actual path and returns a `<source>`).

I've added a test for multiple sources which currently fails but I would pass with the following.

```
{source || children}
```

@teckliew Thoughts?

Reply:
Yep sounds good! Updated with `{source || children}`